assets/javascripts/discourse/templates/upgrade-header.hbs

```diff
@@ -1 +1 @@
-{{upgrade-notice versionCheck=versionCheck}}
+{{upgrade-notice versionCheck=versionCheck}}
```

Comment:
This is using our old 'custom html' system which magically fetches templates with a certain name. It would be good if we can make it use a plugin outlet instead... but maybe that's something for a separate PR.